chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
-                        g2t_log(
-                            "chrome_restore: JSON parse failed! Error: " +
-                                JSON.stringify(err)
-                        );
-                    }
-                }
-            });
-        } catch (err) {
-            g2t_log("chrome_restore: failed! Error: " + JSON.stringify(err));
-        }
-        return this;
+    const eblcMapID = this.id;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.

(lint/complexity/noThisInStatic)